apps/omg/test/omg/integration/root_chain_coordinator_test.exs

```diff
@@ -29,6 +29,7 @@ defmodule OMG.RootChainCoordinatorTest do
 
   @moduletag :integration
   @moduletag :common
+  @moduletag :capture_log
```

Comment:
would it be possible for the non-alarms-related tests not require this `moduletag`? Maybe the root cause of warnings can be (ideally) fixed or if not, the warnings silenced less globally?

Reply:
I will dig deeper into integrative tests